exo-go/src/docker_setup/index_test.go

```diff
@@ -0,0 +1,74 @@
+package dockerSetup_test
+
+import (
+	"path"
+	"regexp"
+
+	"github.com/Originate/exosphere/exo-go/src/app_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/docker_setup"
+	"github.com/Originate/exosphere/exo-go/src/logger"
+	"github.com/Originate/exosphere/exo-go/src/service_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/types"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("GetServiceDockerConfigs", func() {
+
+	var setup *dockerSetup.DockerSetup
+	var appDir string
+
+	var _ = BeforeSuite(func() {
+		appDir = path.Join("..", "..", "..", "exosphere-shared", "example-apps", "external-dependency")
+		appConfig, err := appConfigHelpers.GetAppConfig(appDir)
+		Expect(err).NotTo(HaveOccurred())
+		serviceConfigs, err := serviceConfigHelpers.GetServiceConfigs(appDir, appConfig)
+		Expect(err).NotTo(HaveOccurred())
+		serviceData := serviceConfigHelpers.GetServiceData(appConfig.Services)
+		serviceName := "mongo"
+		setup = dockerSetup.NewDockerSetup(appConfig, serviceConfigs[serviceName], serviceData[serviceName], serviceName, logger.NewLogger([]string{}, []string{}), appDir)
+	})
+
+	var _ = Describe("GetServiceDockerConfigs", func() {
+		var dockerConfigs map[string]types.DockerConfig
```

Comment:
If this is needed in multiple tests, please create it in a `beforeEach` so the tests do not rely on each other